docs/tutorials/regression/laplace.ipynb

```diff
@@ -319,7 +321,9 @@
    "source": [
     "## Prediction\n",
     "\n",
-    "For prediction we can either rely on the `trainer.test()` method or manually conduct a `predict_step()`. Using the trainer will save the predictions and some metrics to a CSV file, while the manual `predict_step()` with a single input tensor will generate a dictionary that holds the mean prediction as well as some other quantities of interest, for example the predicted standard deviation or quantile. The Laplace wrapper module will conduct the Laplace fitting procedure automatically before making the first prediction and will use it for any subsequent call to sample network weights for the desired number of Monte Carlo samples."
+    "For prediction we can either rely on the `trainer.test()` method or manually conduct a `predict_step()`. Using the trainer will save the predictions and some metrics to a CSV file, while the manual `predict_step()` with a single input tensor will generate a dictionary that holds the mean prediction as well as some other quantities of interest, for example the predicted standard deviation or quantile. The Laplace wrapper module will conduct the Laplace fitting procedure automatically before making the first prediction and will use it for any subsequent call. Originally, this was done through sampling and multiple forward passes, however, [Immer et al. 2021](https://arxiv.org/abs/2008.08400) showed that a linearization of the model achieves better performance in practice:\n",
+    "$f_{\\theta}(x)=f_{\\theta_{MAP}}(x)+ J_{\\theta_{MAP}}(\\theta-\\theta_{MAP})$\n",
+    "and that is the current default implementation. However, arguments can be passed to the class or the individual predict step to choose between the following procedures of sampling as `pred_type=\"glm\"` (linearization) or `pred_type=\"nn\"` (sampling)."
```

Comment:
The theory section at the start of the notebook suggests that sampling is the default, maybe one could mention the GLM default there as well?